src/features/ModelPlan/ModelToOperations/_components/OptionPanel/index.tsx

```diff
@@ -146,15 +146,18 @@ const MTOOptionsPanel = () => {
 
       <Card
         containerProps={{
-          className: 'radius-lg shadow-2 padding-0 margin-0'
+          className: 'shadow-2 padding-0 margin-0',
+          style: {
+            borderTopLeftRadius: '.65rem',
+            borderTopRightRadius: '.65rem'
+          }
         }}
         data-testid="article-card"
         className="margin-top-2"
       >
         <div
-          style={{ paddingTop: '2px', paddingBottom: '2px' }}
           className={classNames(
-            'display-flex flex-justify bg-base-lightest padding-x-3 radius-top-lg bg-base-lighter'
+            'display-flex flex-justify bg-base-lightest padding-x-3 padding-y-2 bg-base-lighter radius-top-lg'
```

Comment:
The padding here is rather large compared to figma, can probably use a quarter of this padding - `padding-y-05`

Reply:
good catch!